test/recipes/70-test_sslmessages.t

```diff
@@ -46,6 +46,8 @@ my $proxy = TLSProxy::Proxy->new(
     [TLSProxy::Message::MT_CERTIFICATE,
         checkhandshake::ALL_HANDSHAKES
         & ~checkhandshake::RESUME_HANDSHAKE],
+    [TLSProxy::Message::MT_SERVER_KEY_EXCHANGE,
+        checkhandshake::EC_HANDSHAKE],
```

Comment:
In #2154 I disable the whole test for no-ec. I think its better than adding lots of complexity to the test to deal with this option.